lighthouse-core/config/default-config.js

```diff
@@ -304,6 +304,7 @@ const defaultConfig = {
     'byte-efficiency/uses-responsive-images',
     'byte-efficiency/efficient-animated-content',
     'dobetterweb/appcache-manifest',
+    'dobetterweb/bloated-libraries',
```

Comment:
any response to my earlier comment on putting this into performance category instead?

> I might be out of the loop on this so forgive me if you've already discussed this with someone else, but I was expecting this to be a performance opportunity rather than a "General Best Practice".
there's a bit more involved in upgrading to an opportunity but it'd be a lot more prominent in the report and quantify the ROI to users for putting in the effort here :)
I see this is a first step PR and opportunity could come in a followup, but then it still feels like it should be a diagnostic in the performance section to me rather than dobetterweb.

Reply:
Sorry, yes good point. I agree this is a performance opportunity since it directly affects the bundle size. Out of curiosity, why `diagnostics` over `load-opportunities`?

Reply:
Oh I absolutely think it ultimately belongs in `load-opportunities` eventually. It's just more work to quantify exactly how many bytes (and from which scripts) these savings apply, so we generally have a pattern of landing an audit into "diagnostics" (which means we cannot yet quantify the savings) and then in a followup PR add the additional logic to quantify the savings turning it into an opportunity :)

Reply:
Ah makes sense. Can you check it out now? I added a column for the potential bytes savings of switching libraries.